lib/ansible/modules/cloud/vmware/vmware_host_package_info.py

```diff
@@ -117,9 +118,11 @@ def main():
         ],
         supports_check_mode=True,
     )
+    if module._name == 'vmware_host_package_facts':
+        module.deprecate("The 'vmware_host_package_facts' module has been renamed to 'vmware_host_package_info'", version='2.13')
 
     vmware_host_package_config = VmwarePackageManager(module)
-    module.exit_json(changed=False, hosts_package_facts=vmware_host_package_config.gather_package_facts())
+    module.exit_json(changed=False, hosts_package_info=vmware_host_package_config.gather_package_info())
```

Comment:
Here you're still renaming a return value.

Reply:
I missed that. Thanks for pointing it out